### ISSUE TITLE
[$10][MSFT-118] Challenges page: The topcoder logo doesn't have an alt text.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2817]
**Steps to Reproduce**
1. Go to the Challenges page: https://www.topcoder.com/challenges. 2. Ensure screen reader is switched on. 3. Press the shortcut 'ctrl + home' to make sure that you are at the top of the page. 4. Press the down arrow. 5. Notice screen reader reads 'link graphic www.topcoder'

**Expected Result**
Screen reader should announce the logo as 'link graphic topcoder logo'

**Actual Result**
Screen reader announces the topcoder logo incorrectly.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.1.1 Non-Text Content
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2029

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096043 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096043 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096043 has been updated - it has been assigned to gets0ul.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by codeMinter** [comment]
@veshu @Oanh-and-only-Oanh https://github.com/topcoder-platform/community-app/issues/2740 already takes care of the requirement in this ticket. Do we still need this?

**Comment by codeMinter** [comment]
@veshu can you please suggest if we would need this, to me it makes no sense, since #2740 aria-label is read as it is <a/> ? 

@lakshmiathreya thoughts ☝️ ?

**Comment by veshu** [comment]
@codeMinter Lets not merge this and close. You can close the merge request.

**Comment by lakshmiathreya** [comment]
Agreed - this is a duplicate of #2740.

**Comment by codeMinter** [comment]
Thanks @veshu @lakshmiathreya - closed the related PR.

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2817 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2817 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2817 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2817 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097336 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097336 has been updated - it has been assigned to gets0ul.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by Oanh-and-only-Oanh** [comment]
it reads as "link to topcoder homepage" using Apple Voiceover on Chrome.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097336<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by Oanh-and-only-Oanh** [comment]
Failed in PROD because when on the Challenge Listings page, reader does not announce the "Topcoder Logo". It should announce as "Topcoder Logo link to Topcoder Homepage". @nithyaasworld

**Comment by nithyaasworld** [comment]
@gets0ul Could you check this please asap and make it announce as "Topcoder Logo link to Topcoder Homepage" ? Thanks.

**Comment by gets0ul** [comment]
@nithyaasworld ok

**Comment by gets0ul** [comment]
@Oanh-and-only-Oanh 
Did you mean when hovering/mouse over the logo?  
Or keyboard navigation?

**Comment by nithyaasworld** [comment]
This ticket is about Keyboard navigation. 

But I am not sure if it should read out the same thing on mouse-hover too. @Oanh-and-only-Oanh Please confirm.

**Comment by nithyaasworld** [comment]
@gets0ul Please provide the fix for Keyboard navigation.

Once @Oanh-and-only-Oanh confirms, you can include mouse-hover if needed.

**Comment by gets0ul** [comment]
@nithyaasworld @Oanh-and-only-Oanh 
Keyboard navigation is already being read out as "Topcoder Homepage".
https://github.com/topcoder-platform/community-app/blob/4f402f6a4e113f961807bd05f47090db7258cef0/src/shared/components/TopcoderHeader/index.jsx#L354

so I guess I'll wait if mouse hover is wanted too.

**Comment by nithyaasworld** [comment]
@gets0ul 

>Failed in PROD because when on the Challenge Listings page, reader does not announce the "Topcoder Logo". It should announce as "Topcoder Logo link to Topcoder Homepage". 

I think @Oanh-and-only-Oanh wants the reader to include "Topcoder Logo" in the current announcement. Currently it only says "Link to Topcoder homepage". The expectation here is "Topcoder Logo Link to Topcoder Homepage"

**Comment by Oanh-and-only-Oanh** [comment]
@nithya is correct. The tabbing and hoverover should both read “topcoder logo link to homepage”

**Comment by crazyk07** [comment]
@gets0ul we will increase payment for this ticket.  Since half of this has already been paid out, we will create another ticket to pay you another $10 upon completion.

**Comment by gets0ul** [comment]
@nithyaasworld @Oanh-and-only-Oanh new PR is submitted.

**Comment by tosha5252** [comment]
announces 'topcoder homepage link', but doesn't state it's the logo

**Comment by lakshmiathreya** [comment]
@tosha5252 this is not as per requirement clearly stated by @Oanh-and-only-Oanh above. Pls mark as QA fail.

**Comment by gets0ul** [comment]
@tosha5252 @lakshmiathreya 
Tested on Windows 10, Chrome, NVDA and JAWS using fix from #3191 
NVDA: https://drive.google.com/file/d/1fz7c_l-wwN06-4LS7qcA-kCZ8IXmtb4I/view
JAWS: https://drive.google.com/file/d/1k48wS7p5qxb94j_KsAzAMThuPLVrUOwn/view

**Comment by tosha5252** [comment]
@gets0ul it says the wrong thing...should be saying “topcoder logo link to homepage”

**Comment by gets0ul** [comment]
@tosha5252 
So please do share what's your setting when testing it?
JAWS or NVDA, by keyboard or mouse over, using fix from my PR or not?

I shared two videos above, have you watch it? Please share yours too so we can check why the result is different.
Thanks.

**Comment by gets0ul** [comment]
@tosha5252 @lakshmiathreya 
After re-read your comment did you mean that it should be "topcoder logo link to homepage", instead of "topcoder logo link to **topcoder** homepage"

So @Oanh-and-only-Oanh which one should we use here?

**Comment by veshu** [comment]
@tosha5252 @lakshmiathreya 

Please clear your cache and try again. Note it is only deployed on test site.

**Comment by tosha5252** [comment]
Verified working on test site

**Comment by tosha5252** [comment]
Verified

Windows 10 env
Browser:  Chrome
